pkg/scalers/azure_eventhub_scaler.go

```diff
@@ -23,6 +23,7 @@ const (
 	defaultEventHubConsumerGroup     = "$Default"
 	defaultEventHubConnectionSetting = "EventHub"
 	defaultStorageConnectionSetting  = "AzureWebJobsStorage"
+	defaultBlobContainer             = ""
```

Comment:
Alternativly, you can just set the default container here to `azure-webjobs-eventhub` then always use `defaultBlobContainer` in `azure_eventhub.go`

Reply:
actually never mind. I just notices the actual url is different.